src/renderer/store/certificates/certificates.saga.ts

```diff
@@ -2,19 +2,38 @@ import { call, apply, all, takeEvery, put, select } from 'typed-redux-saga'
 import { PayloadAction } from '@reduxjs/toolkit'
 
 import { certificatesActions } from './certificates.reducer'
-import { createUserCsr, configCrypto } from '@zbayapp/identity'
+import { createUserCsr, configCrypto, CertFieldsTypes, parseCertificate } from '@zbayapp/identity'
 import electronStore from '../../../shared/electronStore'
 import { actions as identityActions } from '../handlers/identity'
 import { registrationServiceAddress } from '../../../shared/static'
 import notificationsHandlers from '../../store/handlers/notifications'
 import { successNotification } from '../handlers/utils'
 import directMessagesSelectors from '../selectors/directMessages'
 
+const filterCertificates = (certificates: string[]): string[] => {
+  return certificates.filter((hasCertyficate) => {
+    const certFieldsArray = Object.keys(CertFieldsTypes)
+    const parsedCert = parseCertificate(hasCertyficate)
+
+    let isValid = true
+
+    for (let i = 0; i < certFieldsArray.length; i++) {
```

Comment:
Change to 

for (const field of Object.keys(CertFieldsTypes))

Better than enumerating with index